virtme/commands/run.py

```diff
@@ -179,6 +179,13 @@ def make_parser() -> argparse.ArgumentParser:
         metavar="NAME=PATH",
         help="Add a read/write virtio-blk disk.  The device nodes will be /dev/disk/by-id/virtio-virtme_disk_blk_NAME.",
     )
+    g.add_argument(
+        "--shared-path",
+        action="append",
+        default=[],
+        metavar="TAG=PATH",
+        help="Share a direct using virtiofs with the guest.",
```

Comment:
I think a word is missing here. → _Share a direct XXX using (...)_

Reply:
Well, sorry for not making it clear. I'm using this new option with `--root`, so the directory is not available to the VM. I'm trying to test different versions of SLE, with different userspace rootfs, but my test scripts and livepatches are on the host, so I'm using virtiofsd to make them available to the guest.

Regarding this usage, I'm not sure if this is similar to what you proposed. I never used `overlay-rwdir` so I;m not how this can be used ...

Reply:
OK but then `--rodir` might be enough for your case then? Or do you need something faster? (I think it still makes sense to have it, but maybe you don't really need it for your case?)